addons/stock/views/res_config_settings_views.xml

```diff
@@ -47,6 +47,27 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="col-12 col-lg-6 o_setting_box" id="quality_control">
```

Comment:
I just saw this new quality setting uses the same id than the one in mrp. That means the Timer setting (related exclusively to workorders) is visible in the Stock settings
![2020-12-28_16-26](https://user-images.githubusercontent.com/12071695/103225111-88a60300-4929-11eb-8e55-f376a6acd23b.png)
see https://github.com/odoo/enterprise/blob/3d4ed9ce80c0a9c3fec55bbd58967f5383c89dc2/mrp_workorder/views/res_config_settings_view.xml#L15 :)

Reply:
Fixed at https://github.com/odoo/odoo/pull/63974